nux/components/dot-tip/index.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { defer } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, createRef, compose } from '@wordpress/element';
+import { Popover, Button, IconButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { withSelect, withDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export class DotTip extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.popoverRef = createRef();
+	}
+
+	componentDidMount() {
+		if ( this.props.isVisible ) {
+			// Force the popover to recalculate its position on the next frame. This is a
+			// Temporary workaround to fix the tip not appearing next to the inserter
+			// toggle on page load. This happens because the popover calculates its
+			// position before <PostTitle> is made visible, resulting in the position
+			// being too high on the page.
+			defer( () => {
+				const popover = this.popoverRef.current;
+				const popoverSize = popover.updatePopoverSize();
+				popover.computePopoverPosition( popoverSize );
+				popover.focus();
+			} );
+		}
+	}
+
+	render() {
+		const { children, isVisible, hasNextTip, onDismiss } = this.props;
+
+		if ( ! isVisible ) {
+			return null;
+		}
+
+		return (
+			<Popover
+				ref={ this.popoverRef }
+				className="nux-dot-tip"
+				position="middle right"
+				noArrow
+				focusOnMount
+				role="dialog"
+				aria-label={ __( 'Gutenberg tips' ) }
+				onClose={ onDismiss }
+				onClick={ ( event ) => event.stopPropagation() }
```

Comment:
Why do we stop propagation.

Reply:
>Tips are often nested within buttons. We stop propagation so that clicking on a tip doesn't result in the button being clicked.

This reads perfectly as an inline code comment 😄 (We should be cautious about our use of `stopPropagation. This is a reasonable one, though not clear to the future maintainer why it exists)

Reply:
😀 yep, you're right: https://github.com/WordPress/gutenberg/pull/8339